docs/requirements.txt

```diff
@@ -1,6 +1,3 @@
 pygments
--e git+https://github.com/ipython/traitlets.git#egg=traitlets
--e git+https://github.com/jupyter/jupyter_core.git#egg=jupyter_core
--e git+https://github.com/jupyter/jupyter_client.git#egg=jupyter_client
```

Comment:
Thanks, I put client back. Shouldn't the released dependencies get pulled in when this package is installed in the env?